Multisig/UI/ClaimToken/ClaimNotAvailableViewController.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  ClaimNotAvailableViewController.swift
+//  Multisig
+//
+//  Created by Vitaly on 26.07.22.
+//  Copyright © 2022 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class ClaimNotAvailableViewController: UIViewController {
+
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var descriptionLabel: UILabel!
+    @IBOutlet weak var safeDAOLink: HyperlinkButtonView!
+    @IBOutlet weak var forumLink: HyperlinkButtonView!
+    @IBOutlet weak var okButton: UIButton!
+
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        ViewControllerFactory.addCloseButton(self)
+        ViewControllerFactory.makeTransparentNavigationBar(self)
+
+        titleLabel.setStyle(.claimTitle)
+
+        descriptionLabel.setStyle(.secondary)
+
+        safeDAOLink.setText("Safe DAO")
+        safeDAOLink.url = App.configuration.contact.safeDAOURL
```

Comment:
@DmitryBespalov app crashes here because link retrieval fails. Is there some mistake in the configuration?

Reply:
@DmitryBespalov should prod config be adjusted directly on CI server or is it possible to adjust it via jenkins?